src/_cffi_src/openssl/ssl.py

```diff
@@ -582,6 +590,12 @@
 #if CRYPTOGRAPHY_IS_LIBRESSL
 static const long SSL_OP_NO_DTLSv1 = 0;
 static const long SSL_OP_NO_DTLSv1_2 = 0;
+/* it's not really clear why DTLS1_VERSION is defined, but it is. */
+/*
+don't set this to 0,
+SSL_CTX_set_min_proto_version(TLS1_3_VERSION) would silently allow all version
+*/
+static const long DTLS1_2_VERSION = -1;
```

Comment:
This also needs to be included in `_conditional.py`.

Reply:
`SSL_OP_NO_DTLSv1` etc. aren't in `_conditional.py` yet either. Do you want me to add a new section there for everything in this block or is that intentional?

Reply:
This resolves itself by not including the DTLS constants. :-)